pyeasee/site.py

```diff
@@ -54,15 +54,17 @@ def __init__(self, data: Dict[str, Any], site: Any, easee: Any):
         self.site = site
         self.easee = easee
 
-    async def set_dynamic_current(self, currentP1: int, currentP2: int = None, currentP3: int = None):
+ 
+    async def set_dynamic_current(self, currentP1: int, currentP2: int = None, currentP3: int = None, timeToLive: int = None):
```

Comment:
This should be timeToLive = 0 not timeToLive = None, if 0 = behave as before, right?

Reply:
According to my tests, setting timeToLive to 0 or to None has exactly the same effect. So, with this new API-call, the previous behavior can't be achieved.

Reply:
Sure, but that is probably because the easee API has no clue how to interpret "none" as something useful.
If 0 is the known supported value, that is what we should useI do not like to assume how things work like that.

Reply:
You're probably right. It is better to use 0 as the default value in that case.